padglobal/view/closable_embed.py

```diff
@@ -0,0 +1,9 @@
+from padglobal.view.view_state_base import ViewStateBase
+
+
+class ClosableEmbedViewState(ViewStateBase):
+    def __init__(self, original_author_id, menu_type, raw_query, color, view_type, props):
+        super().__init__(original_author_id, menu_type, raw_query)
+        self.color = color
```

Comment:
add a TODO here about making it support color cos i think we could move color to a central location (like dgcog) but yeah let's not worry about it for now